hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/FlinkMergeHandle.java

```diff
@@ -104,6 +104,11 @@ private void deleteInvalidDataFile(long lastAttemptId) {
     }
   }
 
+  @Override
+  protected void createMarkerFile(String partitionPath, String dataFileName) {
+    // no need to create marker file for flink merge handle.
```

Comment:
can we add the reason here as well? just make it easier to read the code

Reply:
Sure, let me update for it.